### PR TITLE
Enable table properties for JDBC connectors

### PR DESCRIPTION
**Pull request by Praveen2112**
Similar to session property we can allow table properties to be configured when creating tables for JDBC based connector

### CODE REVIEW

File: presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcModule.java [hunk 1]
```diff
@@ -85,4 +85,14 @@ public static void bindProcedure(Binder binder, Class<? extends Provider<? exten
     {
         procedureBinder(binder).addBinding().toProvider(type).in(Scopes.SINGLETON);
     }
+
+    public static Multibinder<TablePropertiesProvider> tablePropertiesProviderMultiBinder(Binder binder)
```
Comment (electrum) [inline presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcModule.java:89]: Remove `Multi` from the name for consistency with `bindSessionPropertiesProvider`

Comment (kokosing) [reply presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcModule.java:89]: How about `tablePropertiesBinder`?

File: presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixClient.java [hunk 2]
```diff
@@ -13,6 +13,8 @@
  */
```
Comment (kokosing) [inline presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixClient.java:13]: > Minor cleanup in Phoenix connector

What kind of cleanup? Commit message is ambiguous. Can you please be more specific.

File: presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcModule.java (hunk 1 above)
Comment (Praveen2112) [reply presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcModule.java]: Renamed it as `bindTablePropertiesProvider`

File: presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixClient.java (hunk 2 above)
Comment (Praveen2112) [reply presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixClient.java:13]: Now we pass the table properties via `JdbcClient` (`PhoenixClient`)

File: presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcModule.java (hunk 1 above)
Comment (findepi) [reply presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcModule.java]: We have `JdbcModule#sessionPropertiesProviderBinder`, so `tablePropertiesProviderBinder` would be consistent.

**Review by kokosing — APPROVED** [review]

**Pull request merged**